src/contracts/pods/EigenPod.sol

```diff
@@ -397,8 +397,8 @@ contract EigenPod is Initializable, ReentrancyGuardUpgradeable, EigenPodPausingC
      * @dev Reverts if `amountWei` is not a whole Gwei amount
      */
     function withdrawRestakedBeaconChainETH(address recipient, uint256 amountWei) external onlyEigenPodManager {
-        require(amountWei % GWEI_TO_WEI == 0, AmountMustBeMultipleOfGwei());
```

Comment:
Can we get a unit test on this

Reply:
Already done `testFuzz_withdrawRestakedBeaconChainETH_AmountGweiNotDivisibleByGwei` in `EigenPodUnit.t.sol`.